src/docs/resources/platform-adaptations.md

```diff
@@ -0,0 +1,218 @@
+---
+title: Platform Specific Behaviors and Adaptations
+---
+
+## Adaptation Philosophy
+
+There are generally 3 categories of platform adaptiveness:
+
+1. Things that are behaviors of the OS environment (such as text editing and
+scrolling) and that would be 'wrong' a different behavior took place.
```

Comment:
..and that would be 'wrong' IF a different...

Reply:
Ah thanks!